### PR TITLE
TELCODOCS-271: 4.6 release note changes for MGMT recovery project

### PR DESCRIPTION
**Pull request by aireilly**
Release notes for: https://issues.redhat.com/browse/TELCODOCS-271

### CODE REVIEW

File: release_notes/ocp-4-6-release-notes.adoc [hunk 1]
```diff
@@ -995,6 +1000,11 @@ Your upgrade to {product-title} 4.6 should now no longer be blocked by this feat
 
 The Node Maintenance Operator now validates maintenance requests for master nodes, preventing master (etcd) quorum violation. As a result, master nodes can only be set into maintenance if the `etcd-quorum-guard` pod disruption budget (PDB) allows it. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1826914[*BZ#1826914*])
 
+[id="ocp-4-6-noe-maintenance-operator-cr"]
```
Comment (slintes) [inline release_notes/ocp-4-6-release-notes.adoc:1003]: probably not important, but there is a typo in the id, missing `d` in `node`

**Review by slintes — APPROVED** [review]
one nit inline, otherwise

/lgtm

File: release_notes/ocp-4-6-release-notes.adoc (hunk 1 above)
Comment (aireilly) [reply release_notes/ocp-4-6-release-notes.adoc]: @slintes fixed anyways, thanks.

File: release_notes/ocp-4-6-release-notes.adoc [hunk 2]
```diff
@@ -1012,11 +1017,16 @@ error: the server doesn't have a resource type "operators"
...
-[id="ocp-4-6-noe-maintenance-operator-validation"]
+[id="ocp-4-6-node-maintenance-operator-validation"]
 ==== Node Maintenance Operator now validates maintenance requests
 
 The Node Maintenance Operator now validates maintenance requests for master nodes, preventing master (etcd) quorum violation. As a result, master nodes can only be set into maintenance if the `etcd-quorum-guard` pod disruption budget (PDB) allows it. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1826914[*BZ#1826914*])
 
+[id="ocp-4-6-node-maintenance-operator-cr"]
+==== Set nodes to maintenance mode using NodeMaintenance custom resources
+
+Previously, putting a node into maintenance mode using a `NodeMaintenance` custom resource was not documented. This is now fixed. For more information, see xref:../virt/node_maintenance/virt-setting-node-maintenance.adoc#virt-setting-node-to-maintenance-mode-with-cr_virt-setting-node-maintenance[Setting a node to maintenance mode with a NodeMaintenance custom resource]
```
Comment (bmcelvee) [inline release_notes/ocp-4-6-release-notes.adoc:1028]: Suggested edit: 

s/custom resource/custom resource (CR)

The link works, but it's directed to the top of the assembly. Just want to make sure that is intentional. :)

**Comment by bmcelvee** [comment]
A suggested edit and a question, otherwise LGTM! 

I gave a cursory check through emails, and didn't see any change management notifications for this one either. Since it's modifying GA'd release notes, it should go through the change management process.

**Comment by aireilly** [comment]
> A suggested edit and a question, otherwise LGTM!
> 
> I gave a cursory check through emails, and didn't see any change management notifications for this one either. Since it's modifying GA'd release notes, it should go through the change management process.

Email sent :)

**Comment by aireilly** [comment]
Hi @bmcelvee, this is ready to merge, and I havn't gotten any feedback from the change managment email I sent. Can you merge this please?

**Pull request merged**